### ISSUE TITLE
Create IoT Hub from site

### ISSUE THREAD

**Issue by gcrev93** [great-lakes/project-egypt#2]

**Comment by howlowck** [comment]
[Screenshot](https://xd.adobe.com/view/2ce4d5fa-f18c-4e53-42e9-14766a13849e-5ad5/screen/40801cd0-83e2-4c2f-9be5-7ad6064faa0d/07-Add-Hub)

**Comment by saroup** [comment]
@howlowck  Same comment applies to this task. The Screenshot you have is for linking an iothub that was created in azure.

**Comment by gcrev93** [comment]
This task is in the back log .. and is the functionality of actually creating the hub from the website. not in azure

**Comment by saroup** [comment]
@gcrev93 yup that's my understanding but the screenshot Hao put is for linking an already existing iothub.
